app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/CoinItem.kt

```diff
@@ -97,22 +97,24 @@ fun CoinItem(
                     top.linkTo(coinName.bottom)
                     width = Dimension.preferredWrapContent
                 },
-            // TODO: Remove dummy value when work on Integrate.
-            text = stringResource(R.string.coin_currency, "24,209"),
+            text = stringResource(
+                R.string.coin_currency,
+                coinItem.currentPrice.toFormattedString()
+            ),
             color = MaterialTheme.colors.textColor,
             style = Style.semiBold16()
         )
 
         PriceChange(
+            coinItem = coinItem,
```

Comment:
Should we avoid passing the model into this component? 🤔 I see that we only need `priceChangePercentage24hInCurrency` property.

Reply:
@hoangnguyen92dn changeable 👍

Reply:
@hoangnguyen92dn Fixed in https://github.com/nimblehq/jetpack-compose-crypto/pull/27/commits/95a4d72049218676751105b68f09127f25a718f7